### PR TITLE
KCON-16: Mute video when Enable Video Conferencing is disabled

### PR DESCRIPTION
**Pull request by harenson**
If `Enable Video Conferencing` option is disabled (uncheck) for a given conference under the `Options` column when using Conferences application, pause video for all the participants joining this conference.

Depends on:
- https://github.com/2600hz/kazoo-conference/pull/25
- https://github.com/2600hz/kazoo-ecallmgr/pull/97
- https://github.com/2600hz/kazoo-crossbar/pull/200

### CODE REVIEW

File: kazoo_call/src/kapps_call_command.erl [hunk 1]
```diff
@@ -2323,6 +2325,10 @@ conference(ConfId, Mute, Deaf, Moderator, ProfileName, Call) ->
 
 -spec conference(kz_term:ne_binary(), boolean(), boolean(), boolean(), kz_term:ne_binary(), boolean(), kapps_call:call()) -> 'ok'.
 conference(ConfId, Mute, Deaf, Moderator, ProfileName, Reinvite, Call) ->
+    conference(ConfId, Mute, Deaf, Moderator, ProfileName, Reinvite, Call, 'false').
+
+-spec conference(kz_term:ne_binary(), boolean(), boolean(), boolean(), kz_term:ne_binary(), boolean(), kapps_call:call(), boolean()) -> 'ok'.
```
Comment (lazedo) [inline kazoo_call/src/kapps_call_command.erl]: is there a reason for not following the pattern (kapps_call:call() comes in last)  ?

File: kazoo_documents/src/kzd_conferences.erl.src [hunk 2]
```diff
@@ -449,6 +450,18 @@ require_moderator(Doc, Default) ->
 set_require_moderator(Doc, RequireModerator) ->
     kz_json:set_value([<<"require_moderator">>], RequireModerator, Doc).
 
+-spec video(doc()) -> any().
```
Comment (jamesaimonetti) [inline kazoo_documents/src/kzd_conferences.erl.src]: These are not the auto-generated specs. Make sure you use the ci-* targets (see make/ci.mk) to run these updates. I'm adding the commit this time

Comment (harenson) [reply kazoo_documents/src/kzd_conferences.erl.src]: Ok, thanks.

File: kazoo_documents/src/kzd_conferences.erl [hunk 3]
```diff
@@ -474,3 +475,15 @@ on_enter_extension(Doc) ->
 -spec set_on_enter_extension(doc(), kz_term:ne_binary()) -> doc().
 set_on_enter_extension(Doc, Extension) ->
     kz_json:set_value([<<"on_enter_extension">>], Extension, Doc).
+
+-spec video(doc()) -> kz_term:api_boolean().
+video(Doc) ->
```
Comment (jamesaimonetti) [inline kazoo_documents/src/kzd_conferences.erl:480]: Why is the field `video`? Should be `should_join_video_muted` or something to suggest boolean() flag.

Comment (harenson) [reply kazoo_documents/src/kzd_conferences.erl:480]: That is the field's name within the conference document.

Comment (jamesaimonetti) [reply kazoo_documents/src/kzd_conferences.erl:480]: Weird that this is getting added now...okay thanks!

**Pull request merged**